flusher.go

```diff
@@ -57,19 +85,19 @@ func Flush(postMetrics [][]DDMetric) {
 	bodyLength := reqBody.Len()
 	Stats.Gauge("flush.content_length_bytes", float64(bodyLength), nil, 1.0)
```

Comment:
This should become a histogram since we're going to have `n` things doing it, yeah?